src/Vts.Scripting/ForwardSolvers/Demo03FluenceOfRhoAndZ.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace Vts.Scripting.ForwardSolvers;
+
+/// <summary>
+/// Class using the Vts.dll library to demonstrate predicting reflectance as a function of 
+/// radial extent and depth using optical properties from a list of chromophore absorbers 
+/// with their concentrations and a power law scatterer for a range of wavelengths
+/// </summary>
+internal class Demo03FluenceOfRhoAndZ : IDemoScript
+{
+    /// <summary>
+    /// Sample script to demonstrate this class' stated purpose
+    /// </summary>
+    public static void RunDemo(bool showPlots = true)
+    {
+        // Example 03: Evaluate fluence as a function of radial extent and depth using optical properties from a list of
+        // chromophore absorbers with their concentrations and a power law scatterer for a range of wavelengths.
+
+        // Solver type options:
+        // PointSourceSDA,DistributedGaussianSourceSDA, DistributedPointSourceSDA,
+        // MonteCarlo(basic scaled), Nurbs(scaled with smoothing and adaptive binning)
+        var solver = new PointSourceSDAForwardSolver();
+        var rhos = new DoubleRange(start: 0.1, stop: 10, number: 100).AsEnumerable().ToArray(); // range of s-d separations in mm
+        var zs = new DoubleRange(start: 0.1, stop: 10, number: 100).AsEnumerable().ToArray(); // range of depths in mm
+
+        // retrieve desired optical properties, based on spectral data information 
+
+        // create an array of chromophore absorbers, each with a given concentrations
+        var chromophores = new IChromophoreAbsorber[]
+        {
+            new ChromophoreAbsorber(ChromophoreType.HbO2, 70), // molar concentration
+            new ChromophoreAbsorber(ChromophoreType.Hb, 30), // molar concentration
+            new ChromophoreAbsorber(ChromophoreType.H2O, 0.8), // fractional concentration
+        };
+
+        // construct a scatterer
+        var scatterer = new PowerLawScatterer(a: 1.2, b: 1.42);
+        // or: var scatterer = new IntralipidScatterer(volumeFraction: 0.5);
```

Comment:
Commented out code causes code smells but I understand the need for this, one thing I have discovered is removing the semi-colon will remove the "smell". 

If you want to leave it as is though, I understand.